1904-10-31.xml

```diff
@@ -0,0 +1,2512 @@
+                        stables. Omnibus meets all Trains.</p>
+                    <p>Ilustrated Prospectus sent free on appplication</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-nea01">
+                    <p>Three-quarters of a Century's established Reputation</p>
+                    <p>Neave's Food</p>
+                    <p>for Infants, Invalids, and the Aged</p>
+                    <p>"Admirably adapted to the wants of infants and young persons." Sir Chas. A
+                        Cameron, C.B., M.D.</p>
+                    <p>Gold Medal awarded, Woman's Exhibition, London, 1900.</p>
+                    <p>Neave' Food is specially prepared for keeping in hot climates and is packed in
+                        hermetically sealed Tins.</p>
+                    <p>Manufacturers: Josiah R. Neave &amp; Co., Fordingbridge, England.</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-baa01">
+                    <head>BOEHME &amp; ANDERER.</head>
+                    <p>NEXT DOOR TO TURF CLUB, CAIRO.</p>
+                    <p>Printers Stationers Bookbinders.</p>
+                    <p>Dealers in Fancy Goods.</p>
+                    <p><measure type="indexNo">7-8-905</measure></p>
+                </div>
+                <div type="advert" xml:id="deg-ad-jma04">
+                    <p>Numbering Machines and Dating Stamps</p>
+                    <p>J. Margosches</p>
+                    <p>Cairo Bulac Road</p>
+                    <p>Stationary Engraving and Printing Office</p>
+                </div>
+                <cb n="4"/>
+                <div type="advert" xml:id="deg-ad-hum02">
+                    <head>Humber Cars &amp; Cycles</head>
+                    <p>Beeston &amp; Coventry.</p>
+                    <p>PAYMENT £15 AND £1 PER MONTH</p>
+                    <p>EGYPTIAN DEPOT:</p>
+                    <p>C. LOUKAITIS &amp; CO.</p>
+                    <p>GOVERNMENT PURVEYORS</p>
+                    <p>Garage – Boulac Street, Cairo</p>
+                </div>
+                <div type="item">
+                    <head>St. Charles' Boarding and Day School</head>
+                    <p>This institution, lately established, is directed by the Sisters of St.
+                        Charles Barrameo.</p>
+                    <p>It is situated in a quiet and healthy part of the City near Helouan Railway
+                        Station, and is accessible by the Nasrieh and Old Cairo tram lines.</p>
+                    <p>The course of studies comprises the English, German, and French languages,
+                        all branches of useful knowledge, needlework, music and painting.</p>
+                    <p>Terms very moderate.</p>
+                    <p>For further information apply to SISTER SUPERIOR, ST. CHARLES' SCHOOL, 11,
+                        Sharia-el-Kassed, Cairo.</p>
+                    <p>24815-3-2</p>
```

Comment:
These numbers, which are part of the paper's accounting system, can be tagged `<measure type="indexNo">`